examples/common/foo_library/BUILD

```diff
@@ -0,0 +1,16 @@
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "common_foo_library",
+    srcs = [
+        "foo_library.cc",
+    ],
+    hdrs = [
+        "foo_library.h",
+    ],
+    defines = ["BAZEL_BUILD"],
+    deps = [
+        "//api",
+        "//sdk/src/trace",
```

Comment:
Does it have sdk dependency?

Reply:
it needes sdk:headers, just fixed it. thanks